features/discovery/src/main/java/org/opennms/netmgt/discovery/messages/DiscoveryJob.java

```diff
@@ -98,4 +101,25 @@ public String toString() {
                  .add("ranges", m_ranges)
                  .toString();
     }
+    
+    /**
+     * <P>
+     * Returns int : the total task timeout for all ip ranges
+     * </P>
+     */
+    public int calculateTaskTimeout(){
+    	BigInteger taskTimeOut = new BigInteger("0");
```

Comment:
Use the https://docs.oracle.com/javase/7/docs/api/java/math/BigInteger.html#ZERO constant here.